docs/source/stubs.rst

```diff
@@ -174,15 +174,8 @@ Type checkers support cyclic imports in stub files.
 Built-in Generics
 -----------------
 
-PEP 585 [#pep585]_ built-in generics are generally supported, with
-the following exceptions [#ts-4820]_:
-
-* Built-in generics don't work in type aliases.
-* Built-in generics don't work in base classes.
-* ``type`` is not supported.
-* Variable length tuples (``tuple[X, ...]``) are not supported.
-
-In these cases, the appropriate types from ``typing`` must be used.
+PEP 585 [#pep585]_ built-in generics are supported and should be used instead
+of the corresponding types from ``typing``.
 
 Using imports from ``collections.abc`` instead of ``typing`` is
 generally possible and recommended.
```

Comment:
Can we drop the "generally" here too? I don't think there are any exceptions left.

Also, it would be good to have some examples in this section. We can list bad examples like "from typing import Sequence" and "List[int]" and good ones like "from collections.abc import Sequence" and "list[int]".

Reply:
I listed a few examples, although I only used positive examples as in other examples in this section.